packages/feedback/src/FeedbackButton.js

```diff
@@ -23,7 +23,12 @@ const FeedbackButton = ({
     onKeyDown={({ keyCode }) => keyCode === 13 && onClick(icon)}
     {...rest}
   >
-    <Icon data-testid="feedback-icon" name={icon} size={iconSize} />
+    <Icon
+      data-testid="feedback-icon"
+      name={icon}
+      size={iconSize}
+      style={{ margin: '0px' }}
```

Comment:
We can move this inline style object outside the component so that it is not recreated on every render or update to the button component. We are doing something similar with `btnStyles` on line 6

This is a good multi-part article explaining the cause of renders and common fixes https://medium.com/welldone-software/why-did-you-render-mr-big-pure-react-component-2a36dd86996f